### ISSUE TITLE
Single curly bracket in code block not rendering code block

### ISSUE THREAD

**Issue by LorenzMichels** [madskristensen/MarkdownEditor2022#124]
If you put a single opening curly bracket in a code block, (part of) that codeblock will not be rendered no matter if you put a backslash in front of it or not. E.g. \```@page "/category-editor/\\{CategoryId:int}"```

**To Reproduce**
1. Go to visual studio
2. Create a md file
3. Copy this example in there: \```@page "/category-editor/\\{CategroyId:int}"```
4. It won't be correctly rendered

**Expected behavior**
I expect this text in a code block: ```@page "/category-editor/{CategoryId:int}"```

By the way, this editor renders it correctly

**Comment by madskristensen** [comment]
I agree that it doesn't render it correctly, but you are using a fenced code block for a single line code format. You shouldn't use triple backticks, but singles in this case. I'm closing this for now. I you still wish to pursue a fix, please open it with the Markdig repo which is where the rendering is done
